OpenTissue/gpu/image/io/image_read.h

```diff
@@ -1,91 +1,180 @@
-#ifndef OPENTISSUE_GPU_IMAGE_IO_IMAGE_READ_H
-#define OPENTISSUE_GPU_IMAGE_IO_IMAGE_READ_H
 //
 // OpenTissue Template Library
 // - A generic toolbox for physics-based modeling and simulation.
 // Copyright (C) 2008 Department of Computer Science, University of Copenhagen.
 //
 // OTTL is licensed under zlib: http://opensource.org/licenses/zlib-license.php
 //
-#include <OpenTissue/configuration.h>
+#pragma once
 
-#include <OpenTissue/gpu/image/image.h>
-#include <OpenTissue/gpu/image/io/image_il_wrap.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <png.h>
 
 #include <iostream>
 #include <string>
+#include <vector>
+
+#include <OpenTissue/configuration.h>
```

Comment:
Usually, the OT convention is to have most local includes first and more distance includes later.

That means OT files first, 3rd party stuff (bundled or nonbundled) next, and finally system dependent stuff last, like standard header libraries

Reply:
Got it, will revert.